src/cli.js

```diff
@@ -201,7 +211,14 @@ Promise.resolve().then(() => {
   if (!output) { return }
   process.stdout.write(output)
   if (errored) { process.exitCode = 2 }
-}).catch(err => {
-  console.log(err.stack) // eslint-disable-line no-console
-  process.exit(err.code || 1)
+}).catch(({ code, message, stack }) => {
+  const exitCode = code === undefined ? 1 : code
+
+  if (code > 0) {
+    process.stdout.write(`${stack}${EOL}`)
```

Comment:
Also i don't think good idea use `stdout` instead `console.log`. See https://github.com/nodejs/node/blob/master/lib/console.js#L43 . `log` wrapper on stdout and work fine in all `os` without `EOL` package.